### PR TITLE
Creates a Yarn Workspace & utilizes React on the client side

### PR DESCRIPTION
**Pull request by scottymeyers**
I suspect that by utilizing React, we'll be able to tackle all of the desired features quicker, such as searching and sorting of the results.

I'd love some confirmation that this works as expected, is a straight forward experience, and worthwhile!

### CODE REVIEW

**Comment by infinityplusone** [comment]
Looking

**Comment by scottymeyers** [comment]
@infinityplusone make sure to pull in the latest once more.

**Review by infinityplusone — CHANGES_REQUESTED** [review]
First off: "Files changed: 2,085" — Who are you, @jaredcunha?

More importantly, can you add a `.gitignore` back and make sure it includes the `.env` and output files (and anything other stuff that might show up in the root)?

**Comment by scottymeyers** [comment]
> First off: "Files changed: 2,085" — Who are you, @jaredcunha?
> 
> More importantly, can you add a `.gitignore` back and make sure it includes the `.env` and output files (and anything other stuff that might show up in the root)?

That stuff is all located in each of the packages now.

**Comment by infinityplusone** [comment]
> That stuff is all located in each of the packages now.

Saw that, but would still like one at the root for the stupid files like `.DS_Store`.

**Comment by scottymeyers** [comment]
> > That stuff is all located in each of the packages now.
> 
> Saw that, but would still like one at the root for the stupid files like `.DS_Store`.

Done.

**Comment by infinityplusone** [comment]
One other thing I just noticed... The output files for the CLI version are created in `./packages/server/`.

1. That feels like the wrong place for them
2. I can't imagine how someone would figure that location out
3. I think it might make sense to create an output folder (at the root) for them and create filenames based on document ID and execution time
4. The messaging when it's done should tell the user where they are (and what they are called)
5. That folder should be added to the `.gitignore`

File: packages/server/output/NHTSA-2021-0001-output.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+commentOn,commentOnDocumentId,duplicateComments,address1,address2,agencyId,city,category,comment,country,displayProperties,docAbstract,docketId,documentType,email,fax,field1,field2,fileFormats,firstName,govAgency,govAgencyType,objectId,lastName,legacyId,modifyDate,organization,originalDocumentId,pageCount,phone,postedDate,postmarkDate,reasonWithdrawn,receiveDate,restrictReason,restrictReasonType,stateProvinceRegion,submitterRep,submitterRepAddress,submitterRepCityState,subtype,title,trackingNbr,withdrawn,zip,openForComment
```
Comment (infinityplusone) [inline packages/server/output/NHTSA-2021-0001-output.csv]: Pretty sure this file shouldn't be included.

File: packages/server/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (infinityplusone) [inline packages/server/package.json]: Pretty sure this file shouldn't be included.

**Review by infinityplusone — CHANGES_REQUESTED** [review]
Couple files got added that shouldn't be there. Also, need to check for and create the `output` directory if it doesn't exist:
```sh
An error occured while writing JSON Object to File.
[Error: ENOENT: no such file or directory, open '.../co-regulations-gov/output/NHTSA-2021-0001-output.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '.../co-regulations-gov/output/NHTSA-2021-0001-output.json'
}
An error occured while converting the JSON to CSV.
[Error: ENOENT: no such file or directory, open '.../co-regulations-gov/output/NHTSA-2021-0001-output.csv'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '.../co-regulations-gov/output/NHTSA-2021-0001-output.csv'
}

```

**Comment by scottymeyers** [comment]
> Couple files got added that shouldn't be there. Also, need to check for and create the `output` directory if it doesn't exist

But it's in the repo, no? I can't tell because I can't view files changes because its overloaded.

**Comment by infinityplusone** [comment]
> > Also, need to check for and create the `output` directory if it doesn't exist
> 
> But it's in the repo, no? I can't tell because I can't view files changes because its overloaded.

It's in the repo inside of `./packages/server/`, but it *correctly* tries to write to `./output/`, which isn't in the repo.

**Comment by scottymeyers** [comment]
I included it in the main directory, but it looks like it was removed. I cannot tell because I can't see the files changed.

**Pull request merged**